tests/phpunit/FileSystem/SourceFileFilterTest.php

```diff
@@ -37,10 +37,11 @@
 
 use function array_values;
 use Infection\FileSystem\SourceFileFilter;
+use Infection\TestFramework\Coverage\Trace;
 use IteratorIterator;
 use PHPUnit\Framework\TestCase;
 use function Pipeline\take;
-use Symfony\Component\Finder\SplFileInfo;
+use SplFileInfo;
```

Comment:
```suggestion
use Symfony\Component\Finder\SplFileInfo;
```

Reply:
I would actually prefer `SplFileInfo` to be honest: it's like when you have a class coupled to a type, it if works with the type interface it's better to typehint the interface (provided that doesn't complexify things).

In other words: since it doesn't cost anything, I don't see why we should couple `SourceFileFilter` to `Symfony\Component\Finder\SplFileInfo` instead of `SplFileInfo`

Reply:
`\SplFileInfo` is definitely better if we're not calling Symfony's version methods.